test/miner_payment_txn_SUITE.erl

```diff
@@ -198,6 +200,57 @@ self_payment_test(Config) ->
     ok.
 
 
+dependent_payment_test(Config) ->
+    Miners = ?config(miners, Config),
+    Count = 50,
+    Chain = ct_rpc:call(hd(Miners), blockchain_worker, blockchain, []),
+    Ledger = ct_rpc:call(hd(Miners), blockchain, ledger, [Chain]),
+    {ok, Fee} = ct_rpc:call(hd(Miners), blockchain_ledger_v1, transaction_fee, [Ledger]),
+
+    lists:foreach(fun(Miner) ->
+                        PayerAddr = ct_rpc:call(Miner, blockchain_swarm, pubkey_bin, []),
+                        Payee = hd(miner_ct_utils:shuffle(Miners -- [Miner])),
+                        PayeeAddr = ct_rpc:call(Payee, blockchain_swarm, pubkey_bin, []),
+                        {ok, _Pubkey, SigFun, _ECDHFun} = ct_rpc:call(Miner, blockchain_swarm, keys, []),
+                        UnsignedTxns = [ ct_rpc:call(Miner, blockchain_txn_payment_v1, new, [PayerAddr, PayeeAddr, 1, Fee, Nonce]) || Nonce <- lists:seq(1, Count) ],
+                        SignedTxns = [ ct_rpc:call(Miner, blockchain_txn_payment_v1, sign, [Txn, SigFun]) || Txn <- UnsignedTxns],
+                        put(a_txn, {Miner, lists:last(SignedTxns)}),
+                        [ ok = ct_rpc:call(Miner, blockchain_worker, submit_txn, [SignedTxn]) || SignedTxn <- miner_ct_utils:shuffle(SignedTxns) ]
```

Comment:
maybe a straight reverse here to always test the worst case?

Reply:
done